src/features/offer/components/OfferTags/OfferTags.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled from 'styled-components/native'
+
+import { Tag, TAG_HEIGHT } from 'ui/components/Tag/Tag'
+import { getSpacing } from 'ui/theme'
+
+interface Props {
+  tags: string[]
+  tagsLines?: number
+}
+
+export const MARGIN_BOTTOM_TAGS = getSpacing(2)
+
+// This component is used to display tags on a number of lines defined as a parameter.
+// If the number of tags to display exceeds this limit, it will not be visible
+export function OfferTags({ tags, tagsLines = 2 }: Readonly<Props>) {
+  const maxContainerHeight = (TAG_HEIGHT + MARGIN_BOTTOM_TAGS) * tagsLines
```

Comment:
Hm... Je ne pense pas que ton calcul soit bon.
Je pense qu'il doit être 
```suggestion
  const maxContainerHeight = (TAG_HEIGHT * tagsLines) + (MARGIN_BOTTOM_TAGS * (tagsLines - 1))
```

Si on admet que `TAG_HEIGHT = 24` et que `MARGIN_BOTTOM_TAGS = 4` alors on peut convertir :
- si `tagsLines = 0`, alors `maxContainerHeight = 0` (ça n'arrivera jamais)
- si `tagsLines = 1`, alors `maxContainerHeight = 24 (TAG_HEIGHT * 1 /* 24 */ + (MARGIN_BOTTOM_TAGS * 0 /* 0 */)`
- si `tagsLines = 2`, alors `maxContainerHeight = 52 (TAG_HEIGHT * 2 /* 48 */ + (MARGIN_BOTTOM_TAGS * 1 /* 4 */)`
- ...

De cette façon tu ne crées pas de marge qui ne dois pas exister et qui pourrait créer des glitches. Tu pourrais en créer une fonction que tu testes pour t'en assurer peut-être ?